### ISSUE TITLE
Add support for Python

### ISSUE THREAD

**Issue by rijumone** [wj-Mcat/code-generator#31]
Currently, I could not find support to output `python` source code. Need to add that support. If it already exists, then it needs to be added to the documentation.

**Comment by wj-Mcat** [comment]
Yes, this project is the `dynamic template` for `vue` GRUD code, which is my own code. This dynamic template can be customized by developer for his/her own project. So, this `code-generator` is the core template engine not include different project template.

In theory this tool can generate code for all programming languages, if you write your own template script. 

If you have willing, you can contribute your own project template.  Thanks for your attention on this project.
